src/components/LayerControl/LayerControl.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    <div class="d-flex align-center">
+      <span
+        class="sortable-handle py-3"
+        :data-id="id"
+        :data-parent-ids="parentIds"
+      >
+        {{ name }}
+      </span>
+      <div v-if="isLayer" class="d-flex align-center ml-3">
+        <v-btn icon @click.stop="$emit('show-info', id)">
+          <v-icon>
+            mdi-information
+          </v-icon>
+        </v-btn>
+      </div>
+    </div>
+    <v-expand-transition>
+      <div v-if="isLayer && selected">
+        <v-slider
+          v-model="layerOpacity"
+          class="pt-2 pb-5 pr-5"
+          hide-details
+          :ripple="false"
+          :max="1"
+          :step="0.1"
+          label="Opacity"
+          @end="$emit('update-layer-opacity', { id, opacity: layerOpacity })"
+        />
+      </div>
+    </v-expand-transition>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'LayerControl',
+    props: {
+      id: {
+        type: String,
+        required: true,
+      },
+      isLayer: {
+        type: Boolean,
+        required: true,
+      },
+      name: {
+        type: String,
+        required: true,
+      },
+      opacity: {
+        type: Number,
+        default: 1,
+        required: true,
+      },
```

Comment:
I think this should probably be reset (and may be other values) when the layer is 'unselected'. But be careful with conflicting store commits that are trying to access the same layer (both layer selection and opacity change trigger something on a `rasterLayer`